src/components/Actions/Actions.vue

```diff
@@ -305,7 +305,7 @@ export default {
 			 */
 			this.$emit('update:open', this.opened)
 		},
-		closeMenu() {
+		closeMenu(e) {
```

Comment:
`e` is not used, so this change does not seem to be needed (in fact, I would have expected the linter to complain :-P ).